src/aws_credentials_ec2.erl

```diff
@@ -3,40 +3,70 @@
 -module(aws_credentials_ec2).
 -behaviour(aws_credentials_provider).
 
+-define(SESSION_TOKEN_URL,
+        "http://169.254.169.254/latest/api/token/").
+-define(SESSION_TOKEN_TTL_HEADER,
+        "x-aws-ec2-metadata-token-ttl-seconds").
+-define(SESSION_TOKEN_TTL_SECONDS,
+        "21600").
+-define(SESSION_TOKEN_HEADER,
+        "X-aws-ec2-metadata-token").
 -define(CREDENTIAL_URL,
```

Comment:
Note that the current changes on master are broken from the commit that changed the format of this macro.

Reply:
Hi @justincjohnson, I believe you are right. This should be now solved in https://github.com/aws-beam/aws_credentials/pull/17 (just saw this PR), so it's probably a good idea to rebase this PR on top of the latest master to get that fix.